spec/helpers/application_helper_spec.rb

```diff
@@ -1,19 +1,15 @@
 require "spec_helper"
-include ActionView::Helpers::JqueryHelper
-include JsHelper
-include ToolbarHelper
```

Comment:
Oh great!!!  Does rails 5/rspec blow up if we try to globally include helpers in tests?  We should tell others about this because I believe this is the way that several of the helpers were done.